client/src/components/CreatePost/CreatePost.js

```diff
@@ -105,6 +105,7 @@ const Wrapper = ({ onCancel, visible, children }) => {
       footer={null}
       visible={visible && currentStep !== 3}
       destroyOnClose={true}
+      overflow-y: hidden;
```

Comment:
ok can't believe I didn't see this but this is actually CSS code on a react element (JSX) attribute... so it needs to go into the styles, currently making the frontend build fail

Reply:
https://github.com/FightPandemics/FightPandemics/pull/597